src/Microsoft.Diagnostics.EventFlow.Inputs.PerformanceCounter/TrackedPerformanceCounter.cs

```diff
@@ -0,0 +1,77 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.Diagnostics.EventFlow.Configuration;
+using Validation;
+
+namespace Microsoft.Diagnostics.EventFlow.Inputs
+{
+    internal class TrackedPerformanceCounter : IDisposable
+    {
+        private PerformanceCounter counter;
+        private bool disposed;
+        private DateTimeOffset lastAccessedOn;
+        private string lastInstanceName;
+
+        public PerformanceCounterConfiguration Configuration { get; private set; }
+
+        public TrackedPerformanceCounter(PerformanceCounterConfiguration configuration)
+        {
+            Requires.NotNull(configuration, nameof(configuration));
+
+            this.Configuration = configuration;
+            this.lastAccessedOn = DateTimeOffset.UtcNow.Subtract(TimeSpan.FromDays(1));
```

Comment:
Comment added. The lastAccessedOn time is set way in the past so that the data is sampled at startup.